guides/micronaut-oracle-cloud-streaming/chess-game/java/src/test/java/example/micronaut/StaticResourceTest.java

```diff
@@ -0,0 +1,24 @@
+package example.micronaut;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import javax.inject.Inject;
+
+@MicronautTest
+public class StaticResourceTest {
+
+    @Inject
+    @Client("/")
+    HttpClient client;
+
+    @Test
+    void staticResourcesAreExposedAtPublic() {
+        HttpResponse<?> response = client.toBlocking().exchange("/index.html");
+        Assert.assertEquals(HttpStatus.OK, response.status());
```

Comment:
should be `org.junit.jupiter.api.Assertions` and use static import. Same with the Kotlin version

Reply:
Done